fe1-web/src/features/social/components/ChirpCard.tsx

```diff
@@ -135,94 +100,107 @@ const ChirpCard = ({ chirp }: IPropTypes) => {
         duration: FOUR_SECONDS,
       });
     });
-    setDeleteModalIsVisible(false);
   };
 
+  const listStyle = List.getListItemStyles(isFirstItem, isLastItem);
+
+  const actionSheetOptions: ActionSheetOption[] = [];
+  if (isSender && !chirp.isDeleted && isConnected) {
+    actionSheetOptions.push({
+      displayName: STRINGS.social_media_delete_chirp,
+      action: deleteChirp,
+    });
+  }
+
   return (
-    <>
-      <View style={styles.container}>
-        <View style={styles.innerContainer}>
-          <View style={styles.leftView}>
-            <ProfileIcon publicKey={chirp.sender} />
-          </View>
-          <View style={styles.rightView}>
-            <View style={styles.senderView}>
-              <Text style={styles.senderText}>{chirp.sender.valueOf()}</Text>
-            </View>
-            {chirp.isDeleted ? (
-              <Text style={styles.deletedChirpText}>{STRINGS.deleted_chirp}</Text>
-            ) : (
-              <Text style={styles.chirpText}>{chirp.text}</Text>
-            )}
-            <View style={styles.reactionsView}>
-              {!chirp.isDeleted && (
-                <>
-                  <View style={styles.reactionView}>
-                    <PoPIconButton
-                      name="thumbsUp"
-                      testID="thumbs-up"
-                      onPress={() => addReaction('👍')}
-                      disabled={addReactionDisabled}
-                    />
-                    <Text>{`  ${thumbsUp}`}</Text>
-                  </View>
-                  <View style={styles.reactionView}>
-                    <PoPIconButton
-                      name="thumbsDown"
-                      testID="thumbs-down"
-                      onPress={() => addReaction('👎')}
-                      disabled={addReactionDisabled}
-                    />
-                    <Text>{`  ${thumbsDown}`}</Text>
-                  </View>
-                  <View style={styles.reactionView}>
```

Comment:
This looks much "worse" than it is, I just used the standard react native elements ListItem structure (https://reactnativeelements.com/docs/4.0.0-rc.6/components/listitem) and moved the existing content into the respective sub-elements.